packages/plugins/plugin-table/src/components/TableContainer.tsx

```diff
@@ -96,7 +96,7 @@ const TableContainer = ({ role, table }: { role?: string; table: TableType }) =>
         }
       }
     },
-    [dispatch, space, schema, table, model, handleThreadCreate, handleInsertRow],
+    [space, schema, table, model, handleThreadCreate, handleInsertRow],
```

Comment:
are space, schema or table used in here either?

Reply:
negative!